packages/nimble-components/src/table/specs/table-columns-hld.md

```diff
@@ -325,69 +325,48 @@ const numberWithUnitCellView = NumberWithUnitCellView.compose({
 
 #### Exposing cell view events externally
 
-<!--
-TODO: Being actively discussed in: https://github.com/ni/nimble/pull/1153#discussion_r1160996644
-
-Finally, here is a column element that allows a user to register a callback for a click event on a button inside the cell template:
+Because the cell view is not a descendant of the column element, we must add special support for the column to handle events that originate in the cells. A column will define a `columnDelegatedEvents` property which is an array of event names that should be delegated. As this is a static property, it will be part of the `ColumnInternalsOptions` passed to the base column constructor.
 
 ```TS
-type TableColumnButtonCellData = TableStringField<'id'>;
-
-public class TableColumnButton extends TableColumn<TableColumnButtonCellData> {
+export interface ColumnInternalsOptions {
     ...
+    readonly columnDelegatedEvents: readonly string[];
```

Comment:
It's a bit redundant to name it columnDelegatedEvents on the columnInternals object. Just `delegatedEvents` is fine

Reply:
Changed